bank_transaction.c

```diff
@@ -0,0 +1,29 @@
+#include <stdio.h>
+
+int main(void)
+{
+int current_balance,transfer_amount;
+int *ptr_current_balance, *ptr_transfer_amount;
+int sum, substact;
+int *new_balance,*new_state;
+int balance;
+
+printf("Please enter amount in source bank account: "); 
+scanf("%d",&current_balance);
+ 
+printf("please enter amount in destination bank account: ");
+scanf("%d",&transfer_amount); 
+
+ptr_transfer_amount= &current_balance; 
+new_balance= &transfer_amount;
+new_state= &balance; 
+
+printf("Please enter amount to transfer: ");
+scanf("%d",&balance); 
+sum = *new_balance + *new_state;
+substact = *ptr_transfer_amount - *new_state;
```

Comment:
so this is really calculating a new balance....?  What happens if the transfer amount is larger than the funds available?  Should that be checked?